api-specificatie/zrc/openapi.yaml

```diff
@@ -2,82 +2,586 @@ openapi: "3.0.0"
 info:
   title: Zaak registratie API
   description: Een API om een zaakregistratiecomponent te kunnen benaderen
-  version: 1.0.0
   contact:
     url:  https://github.com/VNG-Realisatie/gemma-zaken
+  license:
+    name: EUPL 1.2
+  version: 1.0.0
+security:
+  - basic: []
 paths:
-  /zaken:
+  '/betrokkenen/{id}':
```

Comment:
Dit loopt voor op generieke betrokkenen - [polymorfisme](https://swagger.io/docs/specification/data-models/inheritance-and-polymorphism/) is ondersteunt in OpenAPI en ik denk dat we die richting uit gaan moeten om de specialisaties van betrokkenen aan te kunnen bieden.

Reply:
Dit is een hoogst interessant concept...